samwise.js

```diff
@@ -4,23 +4,23 @@
  * Copyright 2016 Henrik Ingo, henrik.ingo@avoinelama.fi
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?